apis/python/notebooks/tutorial_soma_append_mode.ipynb

```diff
@@ -673,6 +686,408 @@
         "    print()\n",
         "    print(X.used_shape())"
       ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "tags": []
+      },
+      "source": [
+        "## Ingesting multiple datasets to a SOMA Experiment"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "tags": []
+      },
+      "source": [
+        "Finally, we'll demonstrate combining multiple new AnnDatas into one experiment.\n",
+        "\n",
+        "The flow is pretty similar to the above:\n",
+        "\n",
+        "1. One call to `register_anndatas` or `register_h5ads` (passing all new AnnDatas/h5ads)\n",
+        "2. One call to `from_anndata`/`from_h5ad` *for each new AnnData*"
+      ]
+    },
+    {
```

Comment:
Add:

Here's a helper function to simulate multiple lab runs. As above, where we used `pbmc3k` to simulate Monday and Tuesday data, here we use `pbmc3k` to simulate multiple AnnData objects.

Reply:
done